packages/web3-providers-ws/src/index.ts

```diff
@@ -0,0 +1,308 @@
+import { EventEmitter } from 'events';
+import {
+	Web3BaseProvider,
+	Web3BaseProviderStatus,
+	JsonRpcId,
+	JsonRpcPayload,
+	JsonRpcResponse,
+	JsonRpcResult,
+	Web3BaseProviderCallback,
+	SubscriptionResultNotification,
+} from 'web3-common';
+import { ClientRequestArgs } from 'http';
+import { ClientOptions, WebSocket, MessageEvent, CloseEvent } from 'isomorphic-ws';
+import {
+	InvalidConnectionError,
+	PendingRequestsOnReconnectingError,
+	ConnectionNotOpenError,
+	InvalidClientError,
+} from 'web3-common/dist/errors';
```

Comment:
Don't import from `dist` folder. 
```suggestion
} from 'web3-common';
```

Reply:
good catch, earlier there was some issue in vs complaining import/export so i used this . now fixed.